dist/restore/index.js

```diff
@@ -0,0 +1 @@
gSeparator(e)}if(this.minimatch.match(e)){return this.trailingSeparator?c.MatchKind.Directory:c.MatchKind.All}return c.MatchKind.None}partialMatch(e){e=i.safeTrimTrailingSeparator(e);if(i.dirname(e)===e){return this.rootRegExp.test(e)}return this.minimatch.matchOne(e.split(l?/\\+/:/\/+/),this.minimatch.set[0],true)}static globEscape(e){return(l?e:e.replace(/\\/g,"\\\\")).replace(/(\[)(?=[^/]+\])/g,"[[]").replace(/\?/g,"[?]").replace(/\*/g,"[*]")}static fixupPattern(e){n(e,"pattern cannot be empty");const t=new u.Path(e).segments.map(e=>Pattern.getLiteral(e));n(t.every((e,t)=>(e!=="."||t===0)&&e!==".."),`Invalid pattern '${e}'. Relative pathing '.' and '..' is not allowed.`);n(!i.hasRoot(e)||t[0],`Invalid pattern '${e}'. Root segment must not contain globs.`);e=i.normalizeSeparators(e);if(e==="."||e.startsWith(`.${o.sep}`)){e=Pattern.globEscape(process.cwd())+e.substr(1)}else if(e==="~"||e.startsWith(`~${o.sep}`)){const t=s.homedir();n(t,"Unable to determine HOME directory");n(i.hasAbsoluteRoot(t),`Expected HOME directory to be a rooted path. Actual '${t}'`);e=Pattern.globEscape(t)+e.substr(1)}else if(l&&(e.match(/^[A-Z]:$/i)||e.match(/^[A-Z]:[^\\]/i))){let t=i.ensureAbsoluteRoot("C:\\dummy-root",e.substr(0,2));if(e.length>2&&!t.endsWith("\\")){t+="\\"}e=Pattern.globEscape(t)+e.substr(2)}else if(l&&(e==="\\"||e.match(/^\\[^\\]/))){let t=i.ensureAbsoluteRoot("C:\\dummy-root","\\");if(!t.endsWith("\\")){t+="\\"}e=Pattern.globEscape(t)+e.substr(1)}else{e=i.ensureAbsoluteRoot(Pattern.globEscape(process.cwd()),e)}return i.normalizeSeparators(e)}static getLiteral(e){let t="";for(let r=0;r<e.length;r++){const n=e[r];if(n==="\\"&&!l&&r+1<e.length){t+=e[++r];continue}else if(n==="*"||n==="?"){return""}else if(n==="["&&r+1<e.length){let n="";let s=-1;for(let t=r+1;t<e.length;t++){const r=e[t];if(r==="\\"&&!l&&t+1<e.length){n+=e[++t];continue}else if(r==="]"){s=t;break}else{n+=r}}if(s>=0){if(n.length>1){return""}if(n){t+=n;r=s;continue}}}t+=n}return t}static regExpEscape(e){return e.replace(/[[\\^$.|?*+()]/g,"\\$&")}}t.Pattern=Pattern},986:function(e,t,r){"use strict";var n=this&&this.__awaiter||function(e,t,r,n){function adopt(e){return e instanceof r?e:new r(function(t){t(e)})}return new(r||(r=Promise))(function(r,s){function fulfilled(e){try{step(n.next(e))}catch(e){s(e)}}function rejected(e){try{step(n["throw"](e))}catch(e){s(e)}}function step(e){e.done?r(e.value):adopt(e.value).then(fulfilled,rejected)}step((n=n.apply(e,t||[])).next())})};var s=this&&this.__importStar||function(e){if(e&&e.__esModule)return e;var t={};if(e!=null)for(var r in e)if(Object.hasOwnProperty.call(e,r))t[r]=e[r];t["default"]=e;return t};Object.defineProperty(t,"__esModule",{value:true});const o=s(r(9));function exec(e,t,r){return n(this,void 0,void 0,function*(){const n=o.argStringToArray(e);if(n.length===0){throw new Error(`Parameter 'commandLine' cannot be null or empty.`)}const s=n[0];t=n.slice(1).concat(t||[]);const i=new o.ToolRunner(s,t,r);return i.exec()})}t.exec=exec}});
```

Comment:
We include these so that we don't need to build as part of the action, correct?

Reply:
I'm not sure you _can_ built as part of the action. Can you?